clients/node/client-rds-data-node/package.json

```diff
@@ -14,17 +14,18 @@
     "test": "exit 0",
     "smoke-test": "npm run pretest && node ./test/smoke/index.spec.js",
     "build:es": "tsc -p tsconfig.es.json",
-    "build:browser": "rollup -c",
```

Comment:
remove rollup from devDependencies in lines 69-72?

Reply:
will do

Reply:
Resolved in https://github.com/aws/aws-sdk-js-v3/pull/404/commits/4387c11f42651b3c5222af1a6fc8ae999e7b58db